solarized-definitions.el

```diff
@@ -316,25 +317,30 @@ the \"Gen RGB\" column in solarized-definitions.el to improve them further."
              (org-hide ((t (,@fg-base03))))
              (org-todo ((t (,@fmt-bold ,@fg-base03 ,@bg-red))))
              (org-done ((t (,@fmt-bold ,@fg-green))))
-             (org-todo-kwd-face ((t (,@fg-red ,@bg-base03))))
-             (org-done-kwd-face ((t (,@fg-green ,@bg-base03))))
+             (org-todo-kwd-face ((t (,@fg-red ,@bg-base03 :box (:line-width 1 :color ,red :style released-button)))))
+             (org-done-kwd-face ((t (,@fg-green ,@bg-base03 :box (:line-width 1 :color ,green :style released-button)))))
              (org-project-kwd-face ((t (,@fg-violet ,@bg-base03))))
-             (org-waiting-kwd-face ((t (,@fg-orange ,@bg-base03))))
-             (org-someday-kwd-face ((t (,@fg-blue ,@bg-base03))))
-             (org-started-kwd-face ((t (,@fg-yellow ,@bg-base03))))
-             (org-cancelled-kwd-face ((t (,@fg-green ,@bg-base03))))
-             (org-delegated-kwd-face ((t (,@fg-cyan ,@bg-base03))))
+             (org-someday-kwd-face ((t (,@fg-magenta ,@bg-base03 :box (:line-width 1 :color ,magenta :style released-button)))))
+             (org-appt-kwd-face ((t (,@fg-orange ,@bg-base03 :box (:line-width 1 :color ,orange :style released-button)))))
+             (org-waiting-kwd-face ((t (,@fg-orange ,@bg-base03 :box (:line-width 1 :color ,orange :style released-button)))))
+             (org-started-kwd-face ((t (,@fg-yellow ,@bg-base03 :box (:line-width 1 :color ,yellow :style released-button)))))
+             (org-next-kwd-face ((t (,@fg-base03 ,@bg-yellow))))
+             (org-cancelled-kwd-face ((t (,@fg-cyan ,@bg-base03 :box (:line-width 1 :color ,cyan :style released-button)))))
+             (org-delegated-kwd-face ((t (,@fg-cyan ,@bg-base03 :box (:line-width 1 :color ,cyan :style released-button)))))
+             (org-deferred-kwd-face ((t (,@fg-cyan ,@bg-base03 :box (:line-width 1 :color ,cyan :style released-button)))))
+             ;; huy: org
+             (org-date ((t (:underline t))))
              ;; table
              (table-cell ((t (,@fmt-none ,@fg-base0 ,@bg-back))))
              ;; outline - pandocBlockQuoteLeader*
-             (outline-1 ((t (,@fmt-none ,@fg-blue))))
-             (outline-2 ((t (,@fmt-none ,@fg-cyan))))
-             (outline-3 ((t (,@fmt-none ,@fg-yellow))))
-             (outline-4 ((t (,@fmt-none ,@fg-red))))
-             (outline-5 ((t (,@fmt-none ,@fg-base0))))
-             (outline-6 ((t (,@fmt-none ,@fg-base01))))
+             (outline-1 ((t (,@fmt-none ,@fg-base3))))
+             (outline-2 ((t (,@fmt-none ,@fg-blue))))
+             (outline-3 ((t (,@fmt-none ,@fg-cyan))))
+             (outline-4 ((t (,@fmt-none ,@fg-green))))
+             (outline-5 ((t (,@fmt-none ,@fg-violet))))
+             (outline-6 ((t (,@fmt-none ,@fg-yellow))))
              (outline-7 ((t (,@fmt-none ,@fg-orange))))
-             (outline-8 ((t (,@fmt-none ,@fg-violet))))
+             (outline-8 ((t (,@fmt-none ,@fg-blue))))
```

Comment:
These faces (at least the first 6) are selected to match the original theme, but again, can be overridden locally once I have #191 merged.